tianshou/env/venvs.py

```diff
@@ -123,7 +120,12 @@ def __getattr__(self, key: str) -> List[Any]:
         wrapped environment, if it does not belong to the wrapping vector
         environment class.
         """
-        return [getattr(worker, key) for worker in self.workers]
+        result = [getattr(worker, key) for worker in self.workers]
+        if key == "action_space":
```

Comment:
I don't know, but can we do better than just check for the single key:"action_space"